src/components/upload/Upload.tsx

```
@@ -110,6 +110,18 @@ export function Upload({
   const fileInputRef = React.useRef<any>()
   const isInputDisabled = fileUploadLimit && files.length >= fileUploadLimit ? true : false
 
+  React.useEffect(() => {
+    console.log(uploadLimitError);
+    
+    const timer = setTimeout(() => {
+      if (!!uploadLimitError.length) {
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)